README.md

```diff
@@ -1,6 +1,7 @@
 # Sway RFCs
 
-
-
 ## Active RFCs
-_None_
+* [Docstrings]
```

Comment:
```suggestion
* [Docstrings]

```

The formatting looks to be broken as it is (see [rendered](https://github.com/FuelLabs/sway-rfcs/blob/720b257cc72f884f42ca6de93f806d11ed788325/README.md)) but I think a newline should fix it?